.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+name: test
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+jobs:
+  unit:
+    name: unit
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1.15.6' # The Go version to download (if necessary) and use.
```

Comment:
Does this action support something more flexible, like maybe just the latest 2 Go releases?  I always found it annoying to Go back and update a hard coded version when there's new patches or releases of Go.

Reply:
I can configure a matrix to run multiple go versions, but as far as I'm aware theres no support for latest or latest - 1